Drivers/cmake/arm-none-eabi.cmake

```diff
@@ -0,0 +1,46 @@
+SET(CMAKE_SYSTEM_NAME Generic)
+set(CMAKE_SYSTEM_PROCESSOR arm)
+include(CMakeForceCompiler)
+set(TOOLCHAIN arm-none-eabi)
+if(NOT DEFINED TOOLCHAIN_PREFIX)
```

Comment:
Could we use find_program here instead to find arm-none-eabi-gcc? Provided the tool is on the path, it should work fine.

Reply:
Sure! The toolchain setup for CMake cross compilation is a bit peculiar, but this makes sense to at least try.

Reply:
Seems to work!